gen/definitions/bgp_peer_address_family.yaml

```diff
@@ -41,15 +41,6 @@ attributes:
     tf_name: control
     type: String
     description: 'Peer address-family control.'
-    enum_values:
```

Comment:
If we remove the enum values we should probably add the possible choices in the description and maybe also provide an example (comma separated list).

Reply:
done